cpp/tests/utilities_tests/column_utilities_tests.cpp

```diff
@@ -68,15 +68,15 @@ TYPED_TEST(ColumnUtilitiesTest, NonNullableToHostWithOffset)
   auto sequence = cudf::test::make_counting_transform_iterator(
     0, [](auto i) { return cudf::test::make_type_param_scalar<TypeParam>(i); });
 
```

Comment:
There should probably be tests of the `fixed_point_column_wrapper`.

Reply:
Tests added.